Taskfile.yml

```diff
@@ -0,0 +1,144 @@
+version: "3"
+
+vars:
+  APP_NAME: node-manager-plugin
+  BIN_DIR: build
+  MAINNET_VERSION: "MAIN.2.5"
+  BUILDNET_VERSION: "DEVN.28.3"
+  NODE_MASSA_DIR: '{{.BIN_DIR}}/node-massa'
+
+tasks:
+  install:
+    cmds:
+      - cmd: go install github.com/go-swagger/go-swagger/cmd/swagger@latest
+
+  generate:
+    cmds:
+      - cmd: mkdir -p ./build
+      - cmd: go generate ./...
+      - task: setup-node-folder
+
+  setup-node-folder:
+    cmds:
+      - cmd: mkdir -p {{.NODE_MASSA_DIR}}
+      - task: setup-node-folder:windows
+      - task: setup-node-folder:linux
+      - task: setup-node-folder:macos 
```

Comment:
No need to have separated tasks.
Just do a single one "setup-node-folder"
And inside it, you put a command for each os using platforms: